lib/Doctrine/DBAL/Schema/Table.php

```diff
@@ -34,7 +34,9 @@ class Table extends AbstractAsset
     protected $_fkConstraints = [];
 
     /** @var mixed[] */
-    protected $_options = [];
+    protected $_options = [
+        'create_options' => [],
```

Comment:
As I already said, associative arrays of associative arrays are barely manageable.